Versions-ios.plist.in

```diff
@@ -55,6 +56,7 @@
 			<string>12.3</string>
 			<string>12.4</string>
 			<string>13.0</string>
+			<string>13.2</string>
```

Comment:
minor (if some other change is required). I know you put it in the PR comments but we're _bound_ to forget at some point. Maybe an XML comment like `<!-- tvOS 13.1 was skipped over to sync with iOS -->`

Reply:
Fair